src/openforms/js/components/admin/form_design/RegistrationFields.js

```diff
@@ -46,14 +47,14 @@ const RegistrationFields = ({
                     />
                 </Field>
             </FormRow>
-            {optionsFormSchema
+            {hasOptionsForm
                 ? (<FormRow>
                     <Field
                         name="form.registrationBackendOptions"
                         label={<FormattedMessage defaultMessage="Registration backend options" description="Registration backend options label" />}
                     >
                         <Form
```

Comment:
Same remark as in the payment tab: the rendering of `Field` and `Form` seems a bit strange:
![Screenshot from 2021-08-17 10-29-10](https://user-images.githubusercontent.com/19154114/129691788-6381a7b8-2de9-4658-8398-e340140e077f.png)